test/dynamo/test_logging.py

```diff
@@ -367,6 +369,83 @@ def fn(a, b):
             if "concrete" in line:
                 self.assertIsNotNone(re.search(r"\(concrete\): \(\d+, \d+\)", line))
 
+    @skipIfNotPy311
+    @make_logging_test(trace_call=True)
+    def test_trace_call(self, records):
+        def fn(x, y):
+            return (x * 2) @ (y * 3)
+
+        fn_opt = torch._dynamo.optimize("eager")(fn)
+        fn_opt(torch.randn(10, 20), torch.randn(20, 30))
+
+        self.assertEqual(len(records), 3)
+        expected = (
+            """\
+            return (x * 2) @ (y * 3)
+                    ~~^~~""",
+            """\
+            return (x * 2) @ (y * 3)
+                              ~~^~~""",
+            """\
+            return (x * 2) @ (y * 3)
+                   ~~~~~~~~^~~~~~~~~""",
+        )
+        for record, answer in zip(records, expected):
+            self.assertIn(answer, record.getMessage())
```

Comment:
ditto here, can use assertExpectedInline(records[0].getMessage(), ...) and so forth

Reply:
I'm not sure using assertExpectedInline for these tests is a good idea here since it would be checking against the entire trace_call log, which includes line number. So the tests would need to be updated every single time code is added above these tests, and the tests also need to be run with EXPECTTEST_ACCEPT=1 in order of definition, which can be different from pytest order. I think the tests are simple enough and I don't anticipate major changes to the expected results.

Reply:
In these cases, you should write a regex to extract out the part you actually want to test against!